routes/users.js

```diff
@@ -1,8 +1,16 @@
 const express = require('express');
 const router = express.Router();
-const { getUsers } = require('../controllers/userController');
+const { getUsers, getUserById, createUser } = require('../controllers/userController');
+
+// POST /user
+router.post('/create', createUser);
 
 // GET /users
-router.get('/', getUsers);
+router.get('/', (req, res) => {
+  if (req.query.id) {
+    return getUserById(req, res);
+  }
+  return getUsers(req, res);
+});
```

Comment:
@igrzi aqui tu não precisa e nem deve fazer assim, melhor fazer desse jeito aqui:

```suggestion
router.get('/', (req, res) => {
  return getUsers(req, res);
});

// Usar o :id que dai ele já até expoem na req, usando req.id (se não me engano 😅, muito tempo sem usar Express não lembro de cabeça ahahahah)
router.get('/:id', (req, res) => {
  return getUserById(req, res)
});
```